docs/HyperIndex/Advanced/effect-api.md

```diff
@@ -145,7 +154,3 @@ Although this solution is very limited, and we're actively working on a better i
 - There might be issues with pulling big caches from the GitHub repository
 
 Join our [Discord](https://discord.gg/envio) to get updates on the progress of the hosted service integration.
-
-### Why Experimental?
-
-The Effect API is currently marked as experimental, but we don't expect major breaking changes in the future. This designation simply means we're actively iterating on the feature and may add new capabilities that could subtly change indexer behavior. We plan to remove the `experimental` tag soon, and your feedback is invaluable in this process!
```

Comment:
Let's replace it `Migrate from Experimental`

1. Remove `experimental_` prefix
2. Add explicit `rateLimit` option. (the experimental has it to false by default)

Also say that it's available starting from 2.32

Reply:
updated and added shipper log video